students/scott/session01/break_me.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+
+# Write a function that generates a NameError
+
+def generate_name_error():
+	print(paul_blart)
+
+generate_name_error()
+
+
+
+
+#Write a function that generates a TypeError
+
+a = 1
+b = "threads"
+
+this_thing = a + b
+
+def generate_type_error():
+	print("this_thing")
+
+generate_type_error()
+
```

Comment:
Note that what is causing your type error is the adding together of two types (1 + 'threads'), and not your function. You need to put the a and b assignments and the addition inside the function to have the error caused by running your generate_type_error function.